Stardrop/Views/MainWindow.axaml.cs

```diff
@@ -1328,7 +1328,7 @@ private async Task HandleStardropUpdateCheck(bool manualCheck = false)
                             string[] arguments = new string[] { $"chmod +x {scriptPath}", $"{scriptPath}" };
                             var processInfo = new ProcessStartInfo
                             {
-                                FileName = "/bin/bash",
+                                FileName = "/bin/env bash",
                                 Arguments = $"-c \"{string.Join(" ; ", arguments)}\"",
```

Comment:
This should be refactored so that `FileName` is `/use/bin/env` and the argument `bash` is specified as an argument, and not as a part of the "file name".

Reply:
Done

Reply:
I would say that the issue that a "file name" is suddenly a file and an argument still prevails. It may work now, but one should be able to presume that a variable `FileName` stores a file name, and not a file name to a program and its argument which needs to be executed in order to get to the actual file name for the bash executable.